### ISSUE TITLE
Unable to package puppeteer ('Chromium revision is not downloaded')

### ISSUE THREAD

**Issue by thatsnoi** [puppeteer/puppeteer#1281]
Hello everyone,
I got the following, which I think is a bug in puppeteer:
I made an application with electron and puppeteer and I've successfully packaged with electron-packager and electron-builder , however when I want to call my puppeteer function, I get this error:

`Assertion failed: Chromium revision is not downloaded. Run "npm install"`

In fact, a few people got the same error with zeit packager, just trying to package puppeteer
-> https://github.com/zeit/pkg/issues/204

Thanks in advance for any help

* Puppeteer version: 0.11.0
* Platform / OS version: Windows 10

**Comment by aslushnikov** [comment]
Hey @thatsnoi, 

Out of curiosity, why would you need to use puppeteer with electron? Can you please share your usecase?

**Comment by thatsnoi** [comment]
Hi @aslushnikov,
Thanks for the answer. I am programming a bot and I am using electron for the UI. You then can input some information and press a button, which basically calls the puppeteer function. After doing that( in the packaged version) I get an error. But as I said, I don't think it has anything to do with electron...

**Comment by thatsnoi** [comment]
Can anyone help me?

**Comment by dodas** [comment]
bump - interested in this as well

**Comment by melkayam92** [comment]
Hey,

My workaround is to give executablePath.
I downloaded chromium separately, then in puppeteer.launch I set the executablePath for the downloaded chromium. 

I packaged the app and it's working good on windows.. but in OS x, I can't seem to launch another executablePath rather than the local. 

I always get "spawn EACCES" when I try run chromium, I checked the permission and it looks fine, any idea how to fix it for mac?
